README.md

```diff
@@ -8,5 +8,5 @@ Twitch/Discord Chat Bot that works for KGB ![monkaS](https://cdn.betterttv.net/e
 
 ```console
 $ cabal v2-build
-$ cabal v2-run secret.json database.db
+$ cabal v2-run kgbotka secret.json database.db
```

Comment:
Is it not `exe:kgbotka`?

Reply:
this apparently works: 

![image](https://user-images.githubusercontent.com/3874017/78451239-b5700700-7684-11ea-8298-d85675eba416.png)

Reply:
Very well then